content/webapp/components/ComicPreviousNext/ComicPreviousNext.tsx

```diff
@@ -0,0 +1,135 @@
+import { FC } from 'react';
+import { ArticleBasic } from '../../types/articles';
+import styled from 'styled-components';
+import Icon from '@weco/common/views/components/Icon/Icon';
+import { chevron } from '@weco/common/icons';
+import Space from '@weco/common/views/components/styled/Space';
+import { font } from '@weco/common/utils/classnames';
+
+const Root = styled.div`
+  position: absolute;
+  z-index: 1;
+  top: 200px;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  overflow: hidden;
+  pointer-events: none;
+
+  ${props => props.theme.media.medium`
+    top: 40%;
+  `}
+`;
+
+const Link = styled.a<{ isNext: boolean }>`
+  position: absolute;
+  color: ${props => props.theme.color('charcoal')};
```

Comment:
nit: I think you need to rebase against main, this and other colours are using the old scheme.